### PR TITLE
remove superflous dashes from Terminus command examples

### LINKED ISSUE #5009: Terminus commands have confusing -- separator
Re: [Terminus Examples](https://pantheon.io/docs/terminus/examples)

Priority: Medium

## Issue Description
Users are reporting issues with commands that are separated with `--` (not as `--flag` but as a command separator)

## Suggested Resolution
Commands still work without the `--` separator. Remove them wherever possible

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #5009

## Effect
PR includes the following changes:
- Mass replace to remove `--` when not needed in Terminus commands.

## Remaining Work
- [ ] List any outstanding work here
- [ ] Technical review
- [ ] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by EdwardAngert** [comment]
after talking with @NeoMeteor, I've learned that the fix in this PR isn't the issue that we had discussed/learned about, so I'm closing this PR for now, and working with him to figure out what docs change is actually necessary.

From what I understand, 
- the `--` is necessary between both drush and wp-cli commands and the command itself.
- the [terminus commands](https://pantheon.io/docs/terminus/commands/) chart has been changed to exclude the `--` needed, both from the command itself and from the option flags.
   - specifically, check out `remote:drush` and `remote:wp`
   - this is going to lead to more user confusion

Additionally, the [commands list](https://pantheon.io/docs/terminus/commands/) shows commands that don't actually exist, and we can't find them in the [terminus repo](https://github.com/pantheon-systems/terminus/tree/master/src/Commands) - specifically `remote:drupal` and `remote:composer`

cc @TeslaDethray

**Comment by EdwardAngert** [comment]
dropping a screenshot of the commands table as Houston had seen it before
this probably all belongs in the issue or in a new issue, but I'm here and need to document it while it's fresh
terminus-remote-drush-wp-command-table

File: source/content/clamav.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ This doc uses the [ClamAV Module for Drupal 7 & 8](https://www.drupal.org/projec
 1. Download and enable the module in the Drupal admin panel, or via [Terminus](/terminus/):
 
    ```bash
-   terminus drush $site.$env -- dl clamav
-   terminus drush $site.$env -- en clamav -y
+   terminus drush $site.$env dl clamav
+   terminus drush $site.$env en clamav -y
```
Comment (TeslaDethray) [inline source/content/clamav.md]: You want the dashes here.

File: source/content/cms-admin.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ Operations that require write access to the codebase must be executed while the
 
  - Activating a new theme in the site admin,
  - Uploading a new module or plugin using an SFTP client,
- - Remote Drush commands, like `terminus remote:drush $site.$env -- pm-enable hsts --yes` <Popover content="Run Drush commands with <a href='/docs/terminus/'>Terminus</a>. For details, see <a href='/docs/drush/'>Drupal Drush Command-Line Utility</a>." />
- - Remote WP-CLI commands, like `terminus remote:wp $site.$env -- plugin install lh-hsts --activate` <Popover content="Run WP-CLI commands with <a href='/docs/terminus/'>Terminus</a>. For details, see <a href='/docs/wp-cli/'>Using WP-CLI On The Pantheon Platform</a>." />
+ - Remote Drush commands, like `terminus remote:drush $site.$env pm-enable hsts --yes` <Popover content="Run Drush commands with <a href='/docs/terminus/'>Terminus</a>. For details, see <a href='/docs/drush/'>Drupal Drush Command-Line Utility</a>." />
```
Comment (TeslaDethray) [inline source/content/cms-admin.md]: All the remote (`remote:drush`, `remote:wp`, `wp`, `drush`) should show the double dashes.

**Pull request closed without merging**